scripts/submodule.sh

```diff
@@ -58,7 +58,7 @@ for i in $(seq 2 $#); do
     (
         cd "${DEFOLD_PATH}/engine/${_SUBMODULE}"
         waf install --platform="${1}" \
-            --prefix="${DYNAMO_HOME}" --skip-codesign --skip-tests --skip-build-tests --opt-level=0
+            --prefix="${DYNAMO_HOME}" --skip-codesign --skip-tests --skip-build-tests --opt-level=0 --with-vulkan
```

Comment:
We should also consider removing "--with-vulkan" from the build scripts, and instead add "--with-opengl"